.github/workflows/main.yml

```diff
@@ -42,8 +42,7 @@ jobs:
           allow-prerelease-opam: true
 
       - run: |
-          opam pin ocaml-protoc 3.0.1 -y -n
-          opam pin pbrt 3.0.1 -y -n
+          opam pin https://github.com/mransan/ocaml-protoc.git#5510694deffde13283742b8ad116fab61b65dfbc -y -n
```

Comment:
fyi a nice way to do this even for testing is using a `pin-depends` field in an *.opam.template file:
https://github.com/semgrep/semgrep/blob/develop/semgrep.opam.template#L17-L20

this gets picked up picked up by dune when it generates the opam file. Additionally, this will also get picked up by nix so you don't have to make the change to flake.nix you have

Reply:
Now that ocaml-protoc 4.0 is released, pinning 4.0 should also work!